audits/internal/README.md

```diff
@@ -0,0 +1,80 @@
+# meme-ooorr
+The review has been performed based on the contract code in the following repository:<br>
+`https://github.com/dvilelaf/meme-ooorr` <br>
+commit: `606acf4ed03c9d66a48e38abe0bce614feaf5ee5` or `tag: v0.0.1-pre-internal-audit` <br>
+
+## Objectives
+The audit focused on Meme* contracts <BR>
+
+### Flatten version
+Flatten version of contracts. [contracts](https://github.com/dvilelaf/meme-ooorr/blob/main/audits/internal/analysis/contracts)
+
+### Security issues
+Details in [slither_full](https://github.com/dvilelaf/meme-ooorr/blob/main/audits/internal/analysis/slither_full.txt) <br>
+All false positive.
+
+## Issue
+## Critical issue
+### scheduleOLASForAscendance() needs Oracle for price
+```
+Needing re-design with TWAP Oracle
+        // TODO: needs oracle integration
+        // Apply 3% slippage protection
+        uint256 limit = _getLowSlippageSafeSwapAmount();
+This is correctly reflected in TODO
+``` 
+### Medium issue
+#### Not check that this LP token doesn't exist, DOS if exist
```

Comment:
This is not an issue. LP CANNOT exist by definition of token not being tradeable by anyone

Reply:
Anyone for DOS can create a pair and revert in https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Factory.sol#L27 IMO